packages/keycloak-user-management/src/components/UserGroupsList/index.tsx

```diff
@@ -159,9 +180,10 @@ export const UserGroupsList: React.FC<UserGroupListTypes> = (props: UserGroupLis
                     overlay={
                       <Menu className="menu">
                         <Menu.Item
+                          key={record.id}
                           className="viewdetails"
                           onClick={() => {
-                            history.push(`${URL_USER_GROUPS}/${record.id}`);
+                            setGroupId(record.id);
```

Comment:
Was there a route already registered in App for this route?

Reply:
@p-netm yes. Didn't remove it because I'm yet to rule out the option of navigation on toggle

Reply:
should we add a comment, maybe in App incase somebody else picks it up.

Reply:
github issue?

Reply:
yeah sure, whichever you fancy

Reply:
@p-netm https://github.com/opensrp/web/issues/972